From 67f32f63d9fb75a0c65647c6367c1c20eb69a74b Mon Sep 17 00:00:00 2001
From: "linfang.zhao" <linfang.zhao@amlogic.com>
Date: Mon, 17 Jul 2023 02:49:43 +0000
Subject: [PATCH 11/15] drm: insmod drm ko crash [1/1]

PD#SWPL-131901

Problem:
using member of null pointer leads to drm ko crash

Solution:
modify debug info

Verify:
SM1

Test:
DRM-OSD-85

Change-Id: I82a2d82bb3691cb4037073296c4491a999b638fe
Signed-off-by: linfang.zhao <linfang.zhao@amlogic.com>
---
 drivers/drm/meson_fbdev.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/drm/meson_fbdev.c b/drivers/drm/meson_fbdev.c
index 93d0b76be..8266be64b 100644
--- a/drivers/drm/meson_fbdev.c
+++ b/drivers/drm/meson_fbdev.c
@@ -737,8 +737,8 @@ int am_meson_drm_fbdev_init(struct drm_device *dev)
 		if (fbdev) {
 			fbdev->zorder = OSD_PLANE_BEGIN_ZORDER + fbdev_cnt;
 			fbdev_cnt++;
-			DRM_INFO("create fbdev(FB:%d) for plane (%d %d %d %s)\n",
-				fbdev->base.fb->base.id, i, osd_plane->plane_index,
+			DRM_INFO("create fbdev for plane (%d %d %d %s)\n",
+				i, osd_plane->plane_index,
 				osd_plane->base.base.id, osd_plane->base.name);
 		} else {
 			DRM_ERROR("create fbdev for plane %d failed\n", i);
-- 
2.25.1

